package.json

```diff
@@ -41,6 +41,202 @@
   ],
   "l10n": "./l10n",
   "contributes": {
+    "walkthroughs": [
+      {
+        "id": "projectExplorer.gettingStarted",
+        "title": "%walkthroughs.gettingStarted.title%",
+        "description": "%walkthroughs.gettingStarted.description%",
+        "featuredFor": [
```

Comment:
@edmundreinhardt To address your question in the old PR, this `featuredFor` field is so that when a workspace contains one of these files, the walkthrough will be featured on the welcome page.

![image](https://github.com/IBM/vscode-ibmi-projectexplorer/assets/32170854/cf56374b-3985-469a-a2bd-2534e51c6826)